apps/price_pusher/src/solana/solana.ts

```diff
@@ -131,47 +134,29 @@ export class SolanaPricePusherJito implements IPricePusher {
       return;
     }
 
-    const transactionBuilder = this.pythSolanaReceiver.newTransactionBuilder({
-      closeUpdateAccounts: false,
-    });
-    await transactionBuilder.addUpdatePriceFeed(
-      priceFeedUpdateData,
-      this.shardId
-    );
-    await transactionBuilder.addClosePreviousEncodedVaasInstructions();
-
-    const transactions = await transactionBuilder.buildVersionedTransactions({
-      jitoTipLamports: this.jitoTipLamports,
-      tightComputeBudget: true,
-      jitoBundleSize: this.jitoBundleSize,
-    });
-
-    const firstSignature = await sendTransactionsJito(
-      transactions.slice(0, this.jitoBundleSize),
-      this.searcherClient,
-      this.pythSolanaReceiver.wallet
-    );
+    for (let i = 0; i < priceIds.length; i += UPDATES_PER_JITO_BUNDLE) {
+      const transactionBuilder = this.pythSolanaReceiver.newTransactionBuilder({
+        closeUpdateAccounts: true,
```

Comment:
Now we can close VAA accounts immediately since the VAA account is not reused in other bundles.